docs/source/storage/discovery_storage.rst

```diff
@@ -1,28 +1,49 @@
 .. _discovery_storage:
 
 *********************************
-Storage Accessible on  Discovery
+Data Storage Options
 *********************************
-There are two main storage systems connected to Discovery: /home and /scratch. These options have specific quotas and limitations.
-The list below details the storage options available to you on Discovery if you have an account on Discovery. These are storage options
-that are connected to Discovery, and you should use when working on Discovery. Every individual with an account on Discovery has
-both a /home and /scratch directory. Research groups can request additional storage on the /work storage system. Note that currently
-/work storage is not provisioned to individuals.
+The Research Computing (RC) team is responsible for the procurement and ongoing maintenance of several data storage options, 
+including active and archive storage solutions. If you are affiliated with Northeastern, you can request one or more storage 
+solutions to meet your storage needs. If you anticipate needing storage as part of a grant requirement, please
+`schedule a storage consultation with a Research Computing staff member <https://rc.northeastern.edu/support/consulting>`_ to understand what
+storage options would best meet your research needs.
+
+Data Storage
+==================================
+
+There are two main storage systems connected to Northeastern's HPC cluster: ``/home`` and ``/scratch``; these options have specific quotas and limitations.
```

Comment:
This says that /home and /scratch have specific quotas and limitations, but in the list below, there isn't a quota listed for /scratch. Should we reword this?

Reply:
I think it would be good to reword because it does sound like there are quotas both on /home and /scratch.